examples/fluent-bit/amazon-managed-service-for-prometheus/README.md

```diff
@@ -0,0 +1,53 @@
+## FireLens Example: Prometheus forward metrics - using the Fluent Bit image with init tag
+
+This example shows how to forward metrics with Fluent Bit's prometheus remote write output plugin to an Amazon Managed Service for Prometheus workspace.
+
+### Step 1: Create config file locally
+
+**fluent-bit.conf**
+
+```
+# FireLens Example: Prometheus forward metrics - using the Fluent Bit image with init tag
+
+# Scape node metrics every 20 seconds (use any metrics input plugin)
```

Comment:
Scrape

Reply:
> (use any metrics input plugin)

What does this mean? This sentence confuses me.

Reply:
Link input docs

Reply:
Updated